src/Scanner.php

```diff
@@ -441,4 +448,35 @@ public function setCheckCertificate($checkCertificate)
     {
         $this->checkCertificate = (bool) $checkCertificate;
     }
+
+    /**
+     * Get verifyHost value
+     * @return int
+     */
+    public function getVerifyHost()
+    {
+        if ($this->checkCertificate) {
+            return 2;
```

Comment:
As per https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYHOST.html, the options aren't a simple binary as with VERIFYPEER, so we transform it slightly here.